lib/sprockets/commoner/bundle.rb

```diff
@@ -63,7 +63,11 @@ def call(input)
         assets_missing = input[:metadata][:commoner_required] - assets_in_bundle
 
         global_identifiers = assets_missing.map do |filename|
-          uri, _ = env.resolve(filename, type: input[:content_type], pipeline: :self, compat: false)
+          uri, _ = if Sprockets::VERSION < '4'
+            env.resolve(filename, accept: input[:content_type], pipeline: :self, compat: false)
+          else
+            env.resolve(filename, accept: input[:content_type], pipeline: :self)
```

Comment:
This will require deprecation notice or another major bump to remove this option later so I think it is better to just do the breaking change in this release.